<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dopusk02_TNR</class>
 <widget class="QMainWindow" name="Dopusk02_TNR">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dopusk02_TNR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="1" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Fill</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="getBtn">
         <property name="text">
          <string>Get</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="setBtn">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Calc sum</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotateBtn">
         <property name="text">
          <string>Rotate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="rotate1Btn">
         <property name="text">
          <string>Rotate &lt;-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="closeBtn">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>400</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>341</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout"/>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>20</y>
         <width>361</width>
         <height>301</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="sizeLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>60</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Size</string>
        </property>
       </widget>
       <widget class="QLabel" name="valueLabel">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>130</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Value</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="sizeSpinBox1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>100</y>
          <width>48</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="sizeSpinBox2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>100</y>
          <width>48</width>
          <height>29</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>12</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="valueSpinBox1">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>170</y>
          <width>48</width>
          <height>29</height>
         </rect>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
       <widget class="QSpinBox" name="valueSpinBox2">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>170</y>
          <width>48</width>
          <height>29</height>
         </rect>
        </property>
        <property name="value">
         <number>2</number>
        </property>
       </widget>
      </widget>
      <widget class="QLineEdit" name="iEdit">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>350</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="jEdit">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>350</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>i =</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>350</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>j =</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>350</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Value</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="valueEdit">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>350</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
